fieldtracing/fieldtracing.h

```diff
@@ -93,10 +94,11 @@ namespace FieldTracing {
    * UNPROCESSED: cells inside the ionosphere or outside the outer limits that weren't even processed in the first place
    */
    enum TracingLineEndType {
+      UNPROCESSED, // Keep first for the reductions to work!
```

Comment:
The only disadvantage of having UNPROCESSED first, is that you can't as easily plot "LOOP" and "UNPROCESSED" together in Visit by simply selecting a range of adjacent numbers. But that's not a major issue anyway.

Reply:
(Also, doesn't changing the order mid-run of FHA make analysis a bit wonky?)

Reply:
Up to 1000 s we had 5% leftover and the plots are ugly anyway. It'll be easy to just not look at that earlier bit of the data. Also, we don't ave access to this, only to the TracingPointConnectionType.